mm-tools/mboot_api.c

```diff
@@ -412,12 +455,22 @@ void mboot_check(struct i2c_drv *iic)
 	}
 }
 
-void mboot(void)
+void mboot(char *mcs_filepath)
 {
-	gmcs1_len = mboot_mcs_file();
+    if (!mcs_filepath) {
+        mcs_filepath = "./mm.mcs"; // Fallback to the hardcoded path if the user doesn't specify the path
+    }
```

Comment:
@AmamiyaRinyuki 
We usually write like this without the brackets if it contains one only.
if (!mcs_filepath)
        mcs_filepath = "./mm.mcs"; // Fallback to the hardcoded path if the user doesn't specify the path